.gitignore

```diff
@@ -11,3 +12,9 @@ logInsatall.log
 symfony/test/functional/node_modules
 symfony/tests/functional/yarn.lock
 coverage/
+
+symfony/test/functional/node_modules
+symfony/test/functional/*.log
+symfony/test/functional/cypress/screenshots
+symfony/test/functional/cypress.json
+symfony/test/functional/cypress/fixtures/user.json
```

Comment:
I think we should git ignore admin.json, not the user.json

Reply:
Added admin.json to git ignore